### PR TITLE
Minor readme fixes

### PR DESCRIPTION
**Pull request by ignasi35**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Antora-based Akka Documentation
 This folder contains the sources for parts of the [Akka web site](https://akka.io/akka-microservices-samples).
```
Comment (ignasi35) [inline README.md:24]: I've ignored this one on purpose until we settle on a name

Comment (patriknw) [reply README.md:24]: You can use https://akka.io/akka-platform-guide
That is anyway in site.yml

Comment (ignasi35) [reply README.md:24]:
```suggestion
This folder contains the sources for parts of the [Akka web site](https://akka.io/akka-platform-guide).
```

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ The reference documentation is available at [doc.akka.io](https://doc.akka.io).
 Example code
 ------------
 
-The example projects used in the documentation are located at
+The example application used in the documentation is composed of 3 three microservices. The projects for these services are located at:
 
 * [Shopping cart service in Scala](docs-source/docs/modules/cqrs/examples/shopping-cart-service-scala)
 * [Shopping cart analytics service in Scala](docs-source/docs/modules/cqrs/examples/shopping-analytics-service-scala)
```
Comment (patriknw) [inline README.md:17]: while you are at it, could you also update the renamed `cqrs` directory? It's now `shopping-microservices`. I missed that.

File: docs-source/docs/modules/concepts/pages/index.adoc [hunk 3]
```diff
@@ -11,7 +11,7 @@ sourcing and CQRS.
 * xref:clustering.adoc[]
 
 * xref:event-sourcing.adoc[]
-* xref:shopping-microservices.adoc[]
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/index.adoc:14]: @patriknw while reviewing the `cqrs/` rename I noticed this and reverted it.

Comment (patriknw) [reply docs-source/docs/modules/concepts/pages/index.adoc:14]: good catch, thanks.

@ennru isn't xref validated?

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**